### ISSUE TITLE
Imlplement Like or contains?

### ISSUE THREAD

**Issue by srhise** [thewei/react-native-store#46]
Any plans to implement a contains filter for property values?

**Comment by thewei** [comment]
can you give me an example?

**Comment by srhise** [comment]
Using this for a recipe database and searching for titles where they
contain certain words. Tried to implement it in source code but it's not
working for some reason, doesn't process the new operator.

On Wed, Mar 9, 2016 at 12:49 AM, thewei notifications@github.com wrote:

> can you give me an example?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/thewei/react-native-store/issues/46#issuecomment-194125334
> .

## 

Sean Hise

**Comment by kintsugi** [comment]
@srhise I assume you are referencing something [like this?](https://docs.strongloop.com/display/public/LB/Where+filter#Wherefilter-likeandnlike)

**Comment by srhise** [comment]
Exactly. Is that something you guys can pull off? I was having trouble
getting it working on my end. I did try :)
